x-pack/test/security_solution_cypress/cypress/tasks/prebuilt_rules.ts

```diff
@@ -142,3 +236,306 @@ export const interceptUpgradeRequestToFail = (rules: Array<typeof SAMPLE_PREBUIL
     delay: 500, // Add delay to give Cypress time to find the loading spinner
   }).as('updatePrebuiltRules');
 };
+
+export const assertRuleInstallSuccessToastShown = (rules: Array<typeof SAMPLE_PREBUILT_RULE>) => {
```

Comment:
This helper is actually OK in this file (shouldn't be moved according to my comment for the whole file), but this already exists: in this same file called `assertInstallationSuccess`.

Was there any need in duplicating this and similar helpers? Was it because `assertInstallationSuccess` goes back to the main Rule Management page and does some assertions there? In that case, I think we should use extract that navigation logic out of `assertInstallationSuccess` and abstract it in another helper, and use them sequentially.